jetstream/core/proto/jetstream.proto

```diff
@@ -26,9 +26,6 @@ service Orchestrator {
 }
 
 message DecodeRequest {
-  // Where to load any pre-existing kv cache from.
-  string session_cache = 1;
```

Comment:
In future, we will have prefill cache logic. Is this filed related with prefill cache?

Reply:
This is for history cache originally. We could add prefill cache fields when we implement it.

Reply:
Sounds good!